test/functional/repository/create_test.rb

```diff
@@ -0,0 +1,41 @@
+require File.join(File.dirname(__FILE__), '../test_helper')
+
+describe "create repository" do
+  let(:org_id) { 1 }
+  let(:product_id) { 2 }
+  let(:name) { "repo1" }
+  let(:content_type) { "yum" }
+
+  it 'with basic options' do
+    api_expects(:repository, :create, 'create a repository') do
+      with_params(
+        'product_id' => product_id,
+        'content_type' => content_type,
+        'name' => name)
+    end
+
+    command = %W(repository create --organization-id #{org_id} --product-id #{product_id}
+                 --content-type #{content_type} --name #{name})
+
+    assert_equal(0, run_cmd(command).exit_code)
+  end
+
+  it 'with ssl options by name' do
+    api_expects(:repository, :create, 'create a repository') do
+      with_params(
+        'organization_id' => org_id,
+        'product_id' => product_id,
+        'content_type' => content_type,
+        'ssl_ca_cert_id' => 1,
+        'ssl_client_cert_id' => 2,
+        'ssl_client_key_id' => 3,
```

Comment:
I suspect the test is failing because one of these ids is a string (i.e. `'1'`) rather than an int. It's difficult to tell though unless we change to using `with_params` as described above.